go/cmd/zk/internal/zkfs/zkfs_test.go

```diff
@@ -0,0 +1,95 @@
+/*
+Copyright 2024 The Vitess Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package zkfs
+
+import (
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/z-division/go-zookeeper/zk"
+)
+
+func TestFS_CopyContext(t *testing.T) {
+	// fs := &FS{}
```

Comment:
Is this a testing artifact? Do you want to remove this in the final PR?

Reply:
Yes my bad. Will do.